### PR TITLE
UCP/ADDRESS: Support packing/unpacking of multiple endpoint addresses

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/ucp/wireup/address.c [hunk 1]
```diff
@@ -628,24 +627,48 @@ static ucs_status_t ucp_address_do_pack(ucp_worker_h worker, ucp_ep_h ep,
...
+                                                           ep_addr_len);
+
+                    /* pack ep address */
+                    status = uct_ep_get_address(ep->uct_eps[lane], ptr);
+                    if (status != UCS_OK) {
+                        return status;
+                    }
+
+                    ucp_address_memcheck(context, ptr, ep_addr_len, rsc_index);
+                    ptr            = UCS_PTR_BYTE_OFFSET(ptr, ep_addr_len);
```
Comment (hoopoepg) [inline src/ucp/wireup/address.c]: strange alignment of '='

File: src/ucp/wireup/address.c [hunk 2]
```diff
@@ -864,23 +888,27 @@ ucs_status_t ucp_address_unpack(ucp_worker_t *worker, const void *buffer,
             ptr       = ucp_address_unpack_length(worker, flags_ptr, ptr,
                                                   &iface_addr_len, 0, &last_tl);
             address->iface_addr = (iface_addr_len > 0) ? ptr : NULL;
+            address->num_ep_addrs = 0;
```
Comment (hoopoepg) [inline src/ucp/wireup/address.c]: unaligned '='

File: src/ucp/wireup/address.c [hunk 3]
```diff
@@ -864,23 +888,27 @@ ucs_status_t ucp_address_unpack(ucp_worker_t *worker, const void *buffer,
...
-                                                             1, &last_ep_addr);
-                address->ep_addr = ptr;
-                ptr              = UCS_PTR_BYTE_OFFSET(ptr, ep_addr_len);
-            } else {
-                address->ep_addr = NULL;
+            last_ep_addr = !(*(uint8_t*)flags_ptr & UCP_ADDRESS_FLAG_HAVE_EP_ADDR);
+            while (!last_ep_addr) {
+                ucs_assert(address->num_ep_addrs < UCP_MAX_LANES);
+                ep_addr       = &address->ep_addrs[address->num_ep_addrs++];
+
```
Comment (hoopoepg) [inline src/ucp/wireup/address.c]: extra spaceline?

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12431/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 29 of 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 02-Oct-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/ca50a1f236233aef6191376bb72dd4e08ca796e2/_main) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/19d1c24cf3a711564c275f2da6afd07a69ca4d97/hpc-arm-cavium-jenkins_W0) | :x: FAILURE
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/7fabff08ccf6d13aada178a87e03a3ff85d8361b/hpc-arm-cavium-jenkins_W1) | :x: FAILURE
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/05f004aa082f1ab2e7326ef7ac74cc53b1c77027/hpc-arm-cavium-jenkins_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/17cfc90a977f22c0a459ef72b821ba530a460d6c/hpc-arm-cavium-jenkins_W3) | :x: FAILURE
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/49d5f61ec88440e04cb9d07ddddfde630f635ce4/hpc-arm-hwi-jenkins_W0) | :x: FAILURE
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/72e53a3e644bfd5c3985391495f856e17fad20f6/hpc-arm-hwi-jenkins_W1) | :x: FAILURE
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/e6b00640a10156dd0553261311d3c5e75ff5f04d/hpc-arm-hwi-jenkins_W2) | :x: FAILURE
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/beee7ceaa788b89c39af7dbd3be0d324fbe8958c/hpc-arm-hwi-jenkins_W3) | :x: FAILURE
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/579046f174224be7a951a7d7fad1b2542374a291/hpc-test-althca_W0) | :x: FAILURE
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/ecd8296a267846a4edda62b23eb2ae2cbf4d52a8/hpc-test-althca_W1) | :x: FAILURE
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/c6327bdc33f2ef318c6ecb58095b85082fe47d72/hpc-test-althca_W2) | :x: FAILURE
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/0ab6ee35336ea9b7b30b8cbcc085787b590e332d/hpc-test-althca_W3) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/75293bbc5fd325014d2e2e5cd2ae594f256ff412/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/d55ae89beced46d96b9b65ca92c756eba828bdcc/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/7466bbd5e7cb7c9a1ae9e3be4d493e2f3b33d1c9/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/5be2f549e12cf51a9c172bacce3498680244937d/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/d385f5bde75dbc45cb39cb537fa86f478b7a58e1/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/c5d7f595b39ae5e5ebe62960ee2d0eeb0fbc89a0/hpc-test-node-legacy_W1) | :x: FAILURE
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/a75ad157f4b16f4a2b8e8b9db20ea1df73c68df1/hpc-test-node-legacy_W2) | :x: FAILURE
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/33675a7ce26c0bf2a788d19d09dd8f0923cebdf9/hpc-test-node-legacy_W3) | :x: FAILURE
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/3618190ed7d716f7b0be0efdd82cd6ac86d6ce5e/hpc-test-node-new_W0) | :x: FAILURE
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/964d54131d2259e814c2da4a5c46dc10cfaafcf1/hpc-test-node-new_W1) | :x: FAILURE
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/0cca174b01c71273bd9b66d7f668e180c5a8f041/hpc-test-node-new_W2) | :x: FAILURE
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/54a5236821d278e9905d47edb39e9f5c5aaea59d/hpc-test-node-new_W3) | :x: FAILURE
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/2fa25e8341f56b50e8c4a88d02ad54c74445cd8c/r-vmb-ppc-jenkins_W0) | :x: FAILURE
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/2c1f5ca1d254b5856af0b7c76497655c01a751f9/r-vmb-ppc-jenkins_W1) | :x: FAILURE
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/aac2fcb898d38930f95e3e8df50587060cfdd782/r-vmb-ppc-jenkins_W2) | :x: FAILURE
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/a7f6777065f0069e8db4294d669da019/raw/2575b76af765e9dfacc02f0a28ec5b813a43d282/r-vmb-ppc-jenkins_W3) | :x: FAILURE

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12451/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 03-Oct-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/d8c3f9ef928788154a0fc8f24e19815ba2609f65/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/5082617e1121b339de4931b08fcbd76920789538/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/cea2ee0ccdc29f08c98cb5453a90d377b5110c4b/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/4d1f8a57974f24f83dbb651019cfa77844c4e41b/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/9a2a485e766a4aab30887eabc7cd39a9275ba252/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/0b58fd85bd1dc37767d2269c7b2e678a89e8d836/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/156acfe54454c3f73500cbaffc9fd4c5962db7e4/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/24394aafc19db0778b74c9622f751443e28a39c1/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/ef5da3d721a5e30c5ae601515670978e02ec17e9/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/4cc4884608859d0386a097237fd0532663b114f4/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/df49937ae335d86952c8a8b59fde5f6cd709186b/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/da8741ed41c69fd32ff0616f5d9b52312ae75084/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/a3eb418cf3743301ec52a97cdc8d2b7d9c9ba5dd/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/3c473a6bd97212e11709c1c5e28c90c0bd610292/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/284f5597b9c7338aa3467356139d368aedeb907c/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/5b4e54db036a5551cf3261a70d1b0601f4c4d0b4/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/99fd1aad51f2d428250e081e81733ce4234d92cb/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/806dded4d6fbab3b20c64dc03ea00f9916c74059/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/97d26e2f234bef8cb84055489b18874890648c8e/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/bd6a230f95490abbb95e86e32ded34d74808c9c4/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/1166727c2d3a5f32f763c897daf17d30d1783970/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/e6c68016272947af77fc2dc890e6a1bc6e4e926c/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/97cf027cc05bf410e55be909c987142db72659e6/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/b1e13ed5df6f9bb603cbfa09db1135fe3f1eaf8a/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/6bbb6c4a20dd016cb54f5a13f26f290f183b25be/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/32c61c9663cf2b968cefc11bcbf1efee5f395399/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/c261490808837f6ec0aa361eb58428739493bbbe/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/3de43a6b9e2fcbc68ab14e23edf346a778c8f9d5/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/d09465406a4e5866ee61cb36a66b1c03/raw/5ea5fe634907097269125a36b03403c0e0909800/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by brminich** [comment]
failures are relevant
```
[2019-09-26T16:21:41.117Z] /scrap/jenkins/workspace/ucx/contrib/../src/ucp/wireup/address.c: In function ‘ucp_address_do_pack’:

[2019-09-26T16:21:41.117Z] /scrap/jenkins/workspace/ucx/contrib/../src/ucp/wireup/address.c:516: error: ‘ep_flags_ptr’ may be used uninitialized in this function
```

File: src/ucp/wireup/address.h [hunk 4]
```diff
@@ -56,19 +56,23 @@ struct ucp_address_iface_attr {
     ucp_tl_iface_atomic_flags_t atomic;       /* Atomic operations */
 };
 
+typedef struct ucp_address_entry_ep_addr {
```
Comment (brminich) [inline src/ucp/wireup/address.h]: why need a separate struct?

Comment (yosefe) [reply src/ucp/wireup/address.h]: in next PR i will add lane_index to it

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12466/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 05-Oct-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/f4513304bd1bc5075d42a9f63c2e00ad869fe217/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/1b37a2bbd7df84898985158dc724e2c4ec551eb2/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/34d4f9a4844612de4e549a9903ba78af17b8e268/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/6f9023f079106937bfcdc0f0691682f704015295/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/a03d18aa52a67a71c93741dc9acddda9c44e9203/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/809a11db2915cd19112e624db13b33b82772a0b7/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/62a65f78a92ce06dba07373d9e8299a3e3506beb/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/a73b49bacedecafa7cefff8f045b4c1da0e7a526/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/a93d229011c8494b8e10ae2affda966a3fedf90c/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/3cf4a688f8ace2d844190b129ac656cfa82f64b9/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/4ea464d3d8575a5f6edf03eb30567b03716841a7/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/34cd9f506f13c819bf0b4aada718e0bb00b34aea/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/c8024acab0ab7af146fe0bd6a7b60a97ff2c9285/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/ad33bbaed0c949d1a77a9a812d647a3b88a52dad/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/2bd528af4af01faa5b03dfb143f78570911d0b73/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/7a4d697c9406aff0fdd86cca02afb29d66cf2bff/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/3c1b82d77421bb7364e3e6460fd82238f45694a5/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/08026fe377b2ee3677436fa6ba43895720744c9f/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/6fb41e0b4f4e07abd3f547f59502135b38b3ee0f/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/0a58318ad88839b16419e1d94e74342390f85c6f/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/b1c0b28da2a5385723da6513be24e4c2e9a78e95/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/25fb8e184b98d01af5e80f578926d799285f2432/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/e5d0a39526aa299e175b0e0003990937fab98081/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/abab1fe584667e1f66c8a8c014f3eac8e1145089/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/11aa6d521c117af84d960fe662b22de5d132c1eb/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/b9c910f08e4903487b77755a1befbab0d5acb8b4/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/5eaf97965d1c05a0f370db1e1f7ace1165b098a7/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/64a3b1af2fc7834ea72313303944e6fd33a5cbbf/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/07f77cc8b27d4275ef5ed1115037dd58/raw/1bb1993648d8332466397e98925a0c32baf60928/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12612/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:bgate:retest
bot:pipe:retest

**Pull request merged**